### PR TITLE
add kafka forwarder

### PR DESCRIPTION
**Pull request by EvanLjp**
add the Kafka log forwarder, the parts should be tested by e22, which would be added when e2e is built in Satellie.  Has passed the manual test, if possible, please merge first.@kezhenxu94 What do you think of Satellite E2E?

### CODE REVIEW

**Comment by wu-sheng** [comment]
About the e2e, we need @arugal 's OAP implementation ready. And for your case, we need log-kafka implementation.

**Comment by wu-sheng** [comment]
Also, you need the agent plugin implementation. Are you going to submit it? How do you plan to do that?

File: dist/licenses/LICENSE-kafka.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2013 Shopify
```
Comment (kezhenxu94) [inline dist/licenses/LICENSE-kafka.txt:1]: The file name should be `LICENSE-sarama.txt`

**Comment by kezhenxu94** [comment]
> @kezhenxu94 What do you think of Satellite E2E? 

I can help to set up the E2E here after OAP implementation is provided.

**Comment by Humbertzhang** [comment]
I am interest in satellite, I can help to set up the E2E

**Comment by wu-sheng** [comment]
The CIs fail, please recheck.

**Review by wu-sheng — APPROVED** [review]
Generally LGTM.

**Review by surlymo — APPROVED** [review]
LGTM

**Pull request merged**